pstress/pstress-run.conf

```diff
@@ -139,27 +139,43 @@ QUERIES_PER_THREAD=100000
+# For PS-8.0, keyring_component will be used (default behaviour)               #
+# For PS-5.7, keyring_file plugin will be used                                 #
+#                                                                              #
+# To enable keyring_file plugin in PS-8.0, set KEYRING_COMPONENT=0             #
+# To disable encryption completely set ENCRYPTION_RUN=0                        #
 ################################################################################
+ENCRYPTION_RUN=1
 KEYRING_PLUGIN="--early-plugin-load=keyring_file.so --keyring_file_data=keyring"
 KEYRING_COMPONENT=1
 
-###############################################################################
-# 5.6/5.7                                                                     #
-# MYSAFE: See http://bugs.mysql.com/?id=73916 re: testing env setup. MYSAFE   #
-# ensures cleaner/better QA runs                                              #
-###############################################################################
-MYSAFE="--no-defaults --max_allowed_packet=33554432 --maximum-bulk_insert_buffer_size=1M --maximum-join_buffer_size=1M --maximum-max_heap_table_size=1M --maximum-max_join_size=1M --maximum-myisam_max_sort_file_size=1M --maximum-myisam_mmap_size=1M --maximum-myisam_sort_buffer_size=1M --maximum-optimizer_trace_max_mem_size=1M --maximum-preload_buffer_size=1M --maximum-query_alloc_block_size=1M --maximum-query_prealloc_size=1M --maximum-range_alloc_block_size=1M --maximum-read_buffer_size=1M --maximum-read_rnd_buffer_size=1M --maximum-sort_buffer_size=1M --maximum-tmp_table_size=1M --maximum-transaction_alloc_block_size=1M --maximum-transaction_prealloc_size=1M --log-output=none --sql_mode=ONLY_FULL_GROUP_BY"
-
 ################################################################################
 # Number of threads to use. Default: 10.                                       #
 ################################################################################
 THREADS=10
+
+################################################################################
+# Enable this option to start Group Replication runs                           #
+################################################################################
+GRP_RPL=0
+
+################################################################################
+# Set to 1 to make this a multi-node GR run (enables multi-node SQL execution  #
+# instead of using a single node ).                                            #
+# In case, GRP_RPL_CLUSTER_RUN=1 , modify configuration parameters as per      #
+# requirements in pstress-cluster-grp.cfg                                      #
+################################################################################
+GRP_RPL_CLUSTER_RUN=0
+
+################################################################################
+# Default GR configuration file for multi-node pstress runs                    #
```

Comment:
It would be good to have description of conf file variables and values.

Reply:
Sure, I can add a description section in pstress-cluster-grp.conf and pstress-cluster-pxc.conf

Reply:
I mean description for options in [pstress/pstress-cluster-grp.cfg]